packages/contract-call-decoder/src/lib/utils.ts

```diff
@@ -0,0 +1,59 @@
+import { BigNumber } from 'ethers';
+
+function arrayContainsMixedTypeKeys(array): boolean {
```

Comment:
Can you add some documentation to what this does and why

Reply:
Here you displayed also another big problem, I included the entire ethers library while I should have inclueded just the [bignumber package](https://github.com/ethers-io/ethers.js/tree/master/packages/bignumber)